### ISSUE TITLE
Does this broker have a /.well-known/openid-configuration end point?

### ISSUE THREAD

**Issue by uclaeamsavino** [awslabs/aws-amplify-identity-broker#540]
I installed the demo, and I can log in from the app. But what I really want to do is integrate with several 3rd party sites that connect over OIDC to our Cognito instance as the user pool provider. Currently the only way to use Cognito as an OIDC provider is to go through the Cognito Hosted UI, which is not acceptable for one of the apps (magento - ecommerce). It's not very user friendly. 

My hope was that I could plug this in as the OIDC provider instead. But it doesn't seem to have a /.well-known/openid-configuration end point, only /.well-known/jwks.json. All the 3rd party plugins I know of require a .well-known/openid-configuration file to work.

Am I missing something?

**Comment by gabrielbutoeru** [comment]
Hi @uclaeamsavino , news regarding this issue? We are also facing this problem. Our idea is to create a new lambda for the .well-known/openid-configuration endpoint, but the configuration file should contain something that makes sense and we are struggling to understand what it should contain and why it is not already exposed/made available by default.

**Comment by uclaeamsavino** [comment]
No, I think they just didn't implement it, and it seems like they're done with this project.

Here's the spec on the .well_known file: https://openid.net/specs/openid-connect-discovery-1_0.html

Here's an example from AWS Cognito: https://cognito-idp.us-west-2.amazonaws.com/us-west-2_cbitCudrT/.well-known/openid-configuration
